tools/pontoon/ui.md

```diff
@@ -0,0 +1,88 @@
+# Understanding the Pontoon User Interface
+
+## Pontoon Translation Workspace
+
+Pontoon's translation workspace features the projects string list in the sidebar on the left and the editing space on the right. The features in the Pontoon user interface are labeled in the image below:
+
+![Translation Workspace](/assets/images/pontoon/ui/translation_workspace.png)
+
+### Menu Bar
+
+The Menu Bar allows you to navigate between projects without leaving the Translation Workspace.
```

Comment:
We could call it the Main toolbar and also mention the right part it:
* notifications
* profile menu (which includes translation-view specific items like upload/download translations and download TM).